crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -168,36 +196,50 @@ impl NotificationClientBuilder {
     }
 }
 
+/// A notification with its full content.
 pub struct NotificationItem {
+    /// Underlying Ruma event.
+    /// TODO(bnjbvr) can we get rid of this?
```

Comment:
Huh, why?

Reply:
I think I've removed it in another PR / commit, but I didn't understand why the client apps needed the underlying event. The reason is that they're applying different things if the message is an image, or just text, or something else, which I simply hadn't thought of. Can remove this comment in the meanwhile.